### DISCUSSION TITLE
Contributions activity problem

### DISCUSSION THREAD

**Discussion by MGabala** [community/community#83894] [Other Feature Feedback, Questions, & Ideas]
Hello i have been commiting for last 3 days but on github i cannot see it in contributions activity section:

![obraz](https://github.com/community/community/assets/82496006/31809cc1-71e4-4b2f-a7e1-de9a17eca19f)
![obraz](https://github.com/community/community/assets/82496006/f8d58f42-c7f3-40ea-9f14-88e8522e277e)

is there a problem?

**Comment by MuhammadArifinHakim** [comment]
There could be several reasons why your contributions are not appearing on your GitHub activity graph :
1. Ensure that the email address associated with your local Git commits matches the email address associated with your GitHub account.
2. Ensure that you have pushed your commits to the remote repository on GitHub. 

For my case, it's always option number 2

**Reply by MGabala** [reply]
I see them in github so they cannot be in local...
